src/attradd.js

```diff
@@ -44,6 +44,9 @@ Snap.plugin(function (Snap, Element, Paper, glob, Fragment) {
                 name = evnt.substring(evnt.lastIndexOf(".") + 1),
                 a = this.attr(name),
                 atr = {};
+            if (name === "text") {
+                return;
+            }
```

Comment:
The check for the text attribute is reAddon is in these three lines.  The rest of the code changes were related to building from the source.